Assets/HoloToolkit/UX/Scripts/Buttons/Button.cs

```diff
@@ -19,20 +19,38 @@ public abstract class Button : MonoBehaviour, IInputHandler, IPointerSpecificFoc
         /// Current Button State
         /// </summary>
         [Header("Basic Settings")]
+        [SerializeField]
         [Tooltip("Current State of the Button")]
-        public ButtonStateEnum ButtonState = ButtonStateEnum.Observation;
+        private ButtonStateEnum buttonState = ButtonStateEnum.Observation;
```

Comment:
Is this the same ButtonStateEnum used by the AppBar?

Reply:
changes to this file are from previous commit(s) by other developers.
Its not part of the Tooltip commit.

Reply:
You may wish to rebase against the may18_dev branch to eliminate files that you are not modifying. the branch was updated to "final" 2017.2.1.4 changes yesterday.